ParisBNF/abb/BNFabb87.xml

```diff
@@ -0,0 +1,209 @@
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="209">209</measure>
+                                    <dimensions type="outer"/>
+                                    <height unit="mm">375</height>
+                                    <width unit="mm">320</width>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="46">   
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">19th-century script</seg> 
+                                <desc><persName
+                                    ref="PRS1071dAbbadi"/>describes the handwriting as ʻneat and niceʼ<foreign xml:lang="fr">(ʻfine et jolieʼ)</foreign>.    
+                                </desc>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#209v"></locus>
+                                    <desc type="GuestText">According to Conti Rossini,  there is a ʻnote in Amharic, containing testimonial information about an unspecified land purchaseʼ.
+                                    </desc>
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding contemporary="true">                  
+                                <decoNote xml:id="b1" type="Other"><persName
+                                    ref="PRS1071dAbbadi"/> describes the binding as ʻwell-preserved beautiful bindingʼ<foreign xml:lang="fr">(ʻbelle reliure peu uséeʼ).</foreign>. </decoNote>                  
+                                <decoNote xml:id="b7" type="SlipCase">According to <persName
+                                    ref="PRS1071dAbbadi"/>,
+                                    the manuscript has a simple two-part leather case.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                            <origin>
+                                <origDate notBefore="1837" notAfter="1848" source="lettering">19th century</origDate>.  This manuscript was collected by <persName
+                                    ref="PRS1071dAbbadi"/> in Ethiopia.  </origin>
```

Comment:
for persons relevant for the manuscript history (scribes, donors, etc etc) it is always good to add a relevant `role` to the `persName `(here `owner `or `beqeather`)

Reply:
Ok! thank you!